topics/homework/week3/file handling/file_handling_task_1.py

```diff
@@ -0,0 +1,27 @@
+# File Handling
+# Read the file and remove equal lines (if any)
+
+def remove_duplicates(file_path):
+    lines = set()
+    try:
+        with open(file_path, 'r') as file:
```

Comment:
File could be opened for read and write at the same time, no need for two different blocks with different permissions